js/WebView.android.js

```diff
@@ -61,6 +61,7 @@ type State = {|
  */
 class WebView extends React.Component<WebViewSharedProps, State> {
   static defaultProps = {
+    overScrollMode: 0,
```

Comment:
Let's add some constants for these modes:

```
OVER_SCROLL_ALWAYS = 0
OVER_SCROLL_CONTENT = 1
OVER_SCROLL_NEVER = 2
```

So then this would be something like

```js
static OVER_SCROLL_ALWAYS = 0
static OVER_SCROLL_CONTENT = 1
static OVER_SCROLL_NEVER = 2
static defaultProps = {
  overScrollMode: OVER_SCROLL_ALWAYS,
  // ...
```

Reply:
Do you want to expose thoses values to the user ? or are you just trying to make the internal code clearer?

Reply:
Good question -- I feel we need them either way, but exposing to the user seems like a good idea. Your thoughts?

Reply:
What if it was changed to use a string, rather than a number? Like:

```js
    overScrollMode?: 'always' | 'content' | 'never',
```

And on the android side, it converted the strings back to their appropriate numbers

Reply:
I really like that idea @empyrical, and it would fit with some other properties, like this one:

https://github.com/react-native-community/react-native-webview/blob/e70a18466e0fd76a042795d7e76fe725a98006a6/js/WebViewTypes.js#L30